### DISCUSSION TITLE
General engine error when analyzing this game

### DISCUSSION THREAD

**Discussion by BuonGa** [official-pikafish/Pikafish#44] [General]
https://www.mediafire.com/file/cwjaogyik1pb5hn/Engine_false1.pgn/file
If it's really for the community, I hope everyone joins hands to fix that mistake

**Comment by xiangqishare** [comment]
This isn't bug; 
it's a normal occurrence due to the rule that a draw is declared if no pieces are captured within 60 moves (Each side can counted checks for a maximum of 10 moves.). 
For the engine, this game is inevitably a draw. As long as no piece is captured within 60 moves, any move can be made, even if it leads to a checkmate on the 61st move, because the game would already be declared a draw by the 60th move.

**Reply by BuonGa** [reply]
Excuse the topic, I would like to send the difference
![z4600168938053_bf9a0fc498a470469dd40f7dfef5218b](https://github.com/official-pikafish/Pikafish/assets/109723881/ced2829c-84a3-479e-8e36-4e3cd0823d27)

**Reply by xiangqishare** [reply]
The second picture is correct.
60move draw after 炮9平4
